packages/e2e-testing/__tests__/bad-network.test.ts

```diff
@@ -0,0 +1,133 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import {configureEnvVariables} from '@statechannels/devtools';
+import _ from 'lodash';
+import {
+  DBAdmin,
+  overwriteConfigWithDatabaseConnection,
+  Wallet as ChannelWallet,
+  defaultTestConfig
+} from '@statechannels/server-wallet';
+import {Logger} from '@graphprotocol/common-ts';
+
+import {clearExistingChannels, generateAllocationIdAndKeys} from '../src/utils';
+import {RECEIPT_SERVER_DB_NAME, PAYER_SERVER_DB_NAME, PAYER_SERVER_URL} from '../src/constants';
+import {createPaymentServer, createReceiptServer} from '../src/external-server';
+
+import {
+  getChannels,
+  getChannelsForAllocations,
+  setupLogging,
+  successfulPayment,
+  syncChannels
+} from './e2e-utils';
+
+// Setup / Configuration
+jest.setTimeout(100_000);
+
+configureEnvVariables();
+
+const LOG_FILE = `/tmp/bad-network-test-without-chain.log`;
+
+const serverArgs = [
+  `--amountOfWorkerThreads ${process.env.AMOUNT_OF_WORKER_THREADS || 0}`,
+  `--channelsPerAllocation 5`,
+  `--dropIncomingRate 0.03`, // 3% chance incoming messages are dropped
+  `--dropOutgoingRate 0.03`, // 3% chance outgoing messages are dropped
```

Comment:
This will cause a flickering test until:
1.  https://github.com/statechannels/statechannels/pull/3275 is merged
2. the CM includes functionality to sync ongoing objectives.

Do we want to create an intentionally flickering tesT?

Reply:
Addressed by setting `numAllocations` to `0`